.github/workflows/test-generator.yml

```diff
@@ -31,18 +31,24 @@ jobs:
   # Create a pull request.
   test-generator:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: library
     env:
-      GENERATE_DATA_SOURCE: protos/google-cloudevents
+      DATA_SOURCE_CHECKOUT_PATH: protos-source-repo
+      GENERATE_DATA_SOURCE: ../protos-source-repo
```

Comment:
nit: remove env var. Also should use the env var LIBRARY_CHECKOUT_PATH to stay consistent with generator.yml?